cmd/fdroidcl/show.go

```diff
@@ -69,13 +69,7 @@ func printAppDetailed(app fdroidcl.App) {
 	p("Summary          :", "%s", app.Summary)
 	p("Added            :", "%s", app.Added.String())
 	p("Last Updated     :", "%s", app.Updated.String())
-	cur := app.CurApk()
-	if cur != nil {
-		p("Current Version  :", "%s (%d)", cur.VName, cur.VCode)
-	} else {
-		p("Current Version  :", "(no version available)")
-	}
-	p("Upstream Version :", "%s (%d)", app.CVName, app.CVCode)
```

Comment:
Fair enough. I'm fine with dropping "upstream", but I would leave the version code in.